test/gui/shared/steps/steps.py

```diff
@@ -399,14 +400,63 @@ def createFolder(context, foldername, username=None):
     path = join(syncPath, foldername)
     os.makedirs(path)
 
-
+ 
+@When('user "|any|" creates "|any|" files inside the folder "|any|"')
+def step(context, username, filenumber, foldername):
+    syncPath = context.userData['currentUserSyncPath']
+    path = join(syncPath, foldername)
+    snooze(5)
```

Comment:
Add comment for the reason to put snooze here. The reason is probably the same as that in create file. Nevertheless, write the comment as well.

Reply:
Syncing all the created files at once seems to have some problem on syncing, as a result not all created files can be synced. So once a file is synced the process will have to wait unless the next file is created. So here we added a snooze of 5 seconds.

Also, the step ``` When the user waits for the files to sync ``` is not working properly as expected. And probably this PR -> #9442 may solve this issue. So we can’t remove ```snooze(5)``` unless the issue is fixed.